src/segment-loader.js

```diff
@@ -717,6 +717,53 @@ export default class SegmentLoader extends videojs.EventTarget {
     };
   }
 
+  abortRequestEarly_(segment) {
+    const stats = segment.stats;
+    const msSinceFirstByte = Date.now() - stats.firstByteReceived;
+    const RTT = stats.roundTripTime;
```

Comment:
This variable loses some clarity of what it actually is, and `stats.roundTripTime` is only used in one place. Probably better to not use a variable for it.